packages/account/src/Components/poi/idv-status/idv-failed/idv-failed.tsx

```diff
@@ -117,12 +117,23 @@ const IdvFailed = ({
 
     const IDV_NOT_APPLICABLE_OPTION = React.useMemo(() => getIDVNotApplicableOption(), []);
 
+    const renderErrorMessage = (mismatch_status: TIDVErrorStatus) => {
+        switch (mismatch_status) {
+            case IDV_ERROR_STATUS.Expired.code:
+                return 'Your identity document has expired.';
+            case IDV_ERROR_STATUS.Failed.code:
+                return 'We were unable to verify the identity document with the details provided.';
```

Comment:
Do you need to move the messages into the config file?

Reply:
Moved it to a config file